app/models/manageiq/providers/openstack/infra_manager/refresh_parser.rb

```diff
@@ -203,9 +203,8 @@ def parse_host(host, indexed_servers, indexed_resources, cloud_hosts_attributes)
         new_result = {
           :name                 => host_name,
           :type                 => 'ManageIQ::Providers::Openstack::InfraManager::Host',
-          :uid_ems              => uid,
+          :uid_ems              => host.instance_uuid,
```

Comment:
Will this require a data migration?

Reply:
We don't use uid_ems for save_hosts_inventory so the first refresh would update existing host records.

We could parse the ems_ref_obj yaml and update uid_ems in a migration though if you want

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/426/commits/d57f12e7d0df25484eed2ec2c29b3e3ed93be8c5